opta/json_schema.py

```diff
@@ -179,11 +173,14 @@ def _check_module_schemas(write: bool = False) -> None:
                 "module_type": CONFIG_TYPE_ENV
                 if module_registry_dict["environment_module"]
                 else CONFIG_TYPE_SERVICE,
-                "clouds": FOLDER_NAME_TO_CLOUD_LIST[cloud],
+                # "clouds": FOLDER_NAME_TO_CLOUD_LIST[cloud],
+                "clouds": json_schema["opta_metadata"]["clouds"],  # @Quinn ok by you?
```

Comment:
Discussed in slack but posting here for posterity -- this is great! Totally fine with me!